lib/index.ts

```diff
@@ -0,0 +1,86 @@
+import * as fs from "fs";
```

Comment:
For this, I recommend using `plop` as we did in the past (much more maintainable and extensible): https://github.com/zeta-chain/zeta-examples/blob/498e3d51b48263184614c94704251c5839fde8a7/plopfile.js#L1.

Reply:
Yes, I think this code could be refactored to use a library. I've looked into `plop` and it's a good candidate, however, right now there are issues with using it with TS https://github.com/plopjs/plop/issues/297

Created an issue to track the refactor: https://github.com/zeta-chain/toolkit/issues/15

This refactor is a nice to have and shouldn't block the PR, I think.